packages/forma-36-codemod/bin/inquirer-choices.js

```diff
@@ -11,7 +13,18 @@ const PARSER_CHOICES = [
 
 const SETUP_CHOICES = [
   {
-    name: 'migrate-all-components-to-v4: Run all exising codemods',
+    name:
+      'update-package-json: Updates package.json file with correct packages',
+    value: 'update-package-json',
+  },
+  {
+    name:
+      'update-package-json-all-v4: Updates package.json file with correct packages and run all existing v4 codemods',
+    value: 'update-package-json-all-v4',
```

Comment:
It will also remove the imports for v3 css files, right?
Can we add it as well to the `name` (description)? Because for me it was confusing. I had to check the code to be sure that this command will actually do everything.